src/ui/simulator/windows/inspector/property.update.cpp

```diff
@@ -1168,6 +1169,11 @@ namespace Inspector
 
 		if (name == "study.buildingmode")
 		{
+			// Gets menu Configure's scenario builder item
+			auto& mainFrm = *Antares::Forms::ApplWnd::Instance();
+			auto* menu = mainFrm.GetMenuBar();
+			auto* sc_builder_menu_item = menu->FindItem(Antares::Forms::mnIDOptionConfigureMCScenarioBuilder);
```

Comment:
It seems like the scenario builder menu item is not disabled when building mode is turned to Automatic on Ubuntu.
GUI on Ubuntu is not fully validated anyway. Debugger on Ubuntu should be run to see if variable **sc_builder_menu_item** is null or not at this point.